src/builtins/BuiltinTemporal.cpp

```diff
@@ -216,6 +302,22 @@ void GlobalObject::installTemporal(ExecutionState& state)
 
     m_temporalPlainDate->setFunctionPrototype(state, m_temporalPlainDatePrototype);
 
+    m_temporalPlainDate->defineOwnPropertyThrowsException(state, ObjectPropertyName(strings->from),
+                                                          ObjectPropertyDescriptor(new NativeFunctionObject(state, NativeFunctionInfo(strings->from, builtinTemporalPlainDateFrom, 1, NativeFunctionInfo::Strict)), (ObjectPropertyDescriptor::PresentAttribute)ObjectPropertyDescriptor::ConfigurablePresent));
+
+    m_temporalPlainDateTime = new NativeFunctionObject(state, NativeFunctionInfo(strings->lazyDateTime(), builtinTemporalPlainDateTimeConstructor, 3), NativeFunctionObject::__ForBuiltinConstructor__);
+    m_temporalPlainDateTime->setGlobalIntrinsicObject(state);
+
+    m_temporalPlainDateTimePrototype = new PrototypeObject(state);
+    m_temporalPlainDateTimePrototype->setGlobalIntrinsicObject(state, true);
+
+    m_temporalPlainDateTimePrototype->defineOwnProperty(state, ObjectPropertyName(strings->constructor), ObjectPropertyDescriptor(m_temporalPlainDateTime, (ObjectPropertyDescriptor::PresentAttribute)ObjectPropertyDescriptor::ConfigurablePresent));
+
+    m_temporalPlainDateTime->setFunctionPrototype(state, m_temporalPlainDateTimePrototype);
+
+    m_temporalPlainDateTime->defineOwnPropertyThrowsException(state, ObjectPropertyName(strings->from),
+                                                              ObjectPropertyDescriptor(new NativeFunctionObject(state, NativeFunctionInfo(strings->from, builtinTemporalPlainDateTimeFrom, 1, NativeFunctionInfo::Strict)), (ObjectPropertyDescriptor::PresentAttribute)ObjectPropertyDescriptor::ConfigurablePresent));
```

Comment:
Why did you invoke `defineOwnPropertyThrowsException` instead of `defineOwnProperty` here?
IMO `defineOwnProperty` is enough for builtin property initialization.